pkg/features/kube_features.go

```diff
@@ -678,7 +662,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	StorageObjectInUseProtection:   {Default: true, PreRelease: featuregate.GA},
 	SupportPodPidsLimit:            {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.21
 	SupportNodePidsLimit:           {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.21
-	HyperVContainer:                {Default: false, PreRelease: featuregate.Deprecated},
+	HyperVContainer:                {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
This change also needs to be reverted in your PR

Reply:
Ok

Reply:
Done, i revert this entries and did a push. I must done other PR?

Reply:
I used with base the branch master. Is correct?